TA-puppet-alert-orchestrator/README.md

```diff
@@ -91,15 +88,15 @@ Once the add-on has been properly configured you can begin creating new Alerts t
 
 1. Create a Splunk search:
 
-	![Splunk Search](readme/img/splunk_search.png)
+	![Splunk Search](d08e551e-fcea-11ee-83a8-92aa9ede6029.png)
```

Comment:
does it matter that these paths changed - or is this just a github thing

Reply:
This internal README is the one that gets used on Splunkbase so those image paths come from the images we've uploaded on that end.